packages/react/menu/src/Menu.tsx

```diff
@@ -209,15 +211,17 @@ const MenuContent = React.forwardRef((props, forwardedRef) => {
   const { forceMount, ...contentProps } = props;
   const context = useMenuContext(CONTENT_NAME);
   return (
-    <Presence present={forceMount || context.open}>
-      <CollectionSlot>
-        {context.isSubmenu ? (
-          <MenuSubContent {...contentProps} ref={forwardedRef} />
-        ) : (
-          <MenuRootContent {...contentProps} ref={forwardedRef} />
-        )}
-      </CollectionSlot>
-    </Presence>
+    <CollectionProvider>
```

Comment:
I've put this one above `Presence` because `Presence` clones its child and adds a ref to it.